ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/CreateSourceCommand.java

```diff
@@ -125,6 +132,20 @@ KsqlTopic getTopic() {
     return topic;
   }
 
+  private TimestampExtractor getDefaultTimestampExtractor(final KsqlConfig ksqlConfig) {
+    try {
+      final Class defaultTimestampExtractor = (Class) ksqlConfig
+          .getKsqlStreamConfigProps().get(
+              StreamsConfig.DEFAULT_TIMESTAMP_EXTRACTOR_CLASS_CONFIG
+          );
+      
+      return (TimestampExtractor) defaultTimestampExtractor.newInstance();
+    } catch (final Exception e) {
+      // TODO: Need to log invalid timestamp configuration here?
```

Comment:
why not violently fail (e.g. raise exception) here? I think I'd rather have my command fail than do something which wasn't what i defined

Reply:
I forgot to remove that line. I left it as a reminder for me to do something there before submitting the PR. 

Anyway, you're right, I should fail instead of just logging the error. I submitted another PR (this time for 5.3.x branch instead of master) which fails in that case.